core/domain/platform_parameter_registry.py

```diff
@@ -376,7 +376,7 @@ def _to_platform_parameter_model(
     default=(
         'You can change your email preferences via the '
         '<a href="%s%s">Preferences</a> page.' % (
-            feconf.OPPIA_SITE_URL,
+            'http://localhost:8181',
```

Comment:
Let's just drop the first %s and use /preferences here

Reply:
Thanks, updated.